raw_package/static/css/esphome-2.css

```diff
@@ -15,11 +14,52 @@
   --alert-warning-color-bg: #fff5e6;
   --alert-error-color: #d93025;
   --alert-error-color-bg: #faefeb;
-
-  --mdc-theme-primary: #03a9f4;
+  --mdc-theme-primary: #000000;
```

Comment:
Any reason we're changing this to black ?

Reply:
As I removed the override in each button that was set to black, otherwise all button text is blue